test/dotnet.Tests/CommandTests/NuGet/GivenANuGetCommand.cs

```diff
@@ -133,5 +135,51 @@ public void ItHasAWhySubcommand()
                 .And.NotHaveStdErr()
                 .And.HaveStdOutContaining("has the following dependency");
         }
+
+        [Fact]
+        public void ItCanUpdatePackages()
```

Comment:
@dsplaisted @marcpopMSFT I don't have experience with Helix. It looks to me like [this test ran and passed](https://dev.azure.com/dnceng-public/public/_build/results?buildId=1061386&view=ms.vss-test-web.build-test-results-tab&runId=28636502&resultId=100915&paneView=debug).  Do I need to take any action regarding the test failures?